sonar-scanner-engine/src/main/java/org/sonar/scanner/scm/ScmConfiguration.java

```diff
@@ -63,19 +64,15 @@
 
   private ScmProvider provider;
 
-  public ScmConfiguration(InputModuleHierarchy moduleHierarchy, AnalysisMode analysisMode, Configuration settings, ScmProvider... providers) {
+  public ScmConfiguration(InputModuleHierarchy moduleHierarchy, AnalysisMode analysisMode, Configuration settings, ScmProvider[] scmProviders) {
```

Comment:
What's wrong with `ScmProvider... scmProviders` ?

Reply:
Nothing, I will switch back. I had done this bc my first plan was to use both ScmProvider[] and ScmBranchProvider[]